src/__stories__/meter-story.tsx

```diff
@@ -100,6 +110,21 @@ export const MeterStory: StoryComponent<MeterStoryArgs> = ({
                         reverse={reverse}
                         values={values}
                         width={fullWidth ? '100%' : width}
+                        {...(extraContent
+                            ? {
+                                  extra: (
```

Comment:
Why not just `extra={extraContent ?  {} : undefined}`

Reply:
done !!